### PR TITLE
Uplift CAPM3RELEASE to v1.1.1 & CAPIRELEASE to v1.1.3

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

**Review by Rozzii — APPROVED** [review]
LGTM

**Review by Sunnatillo — COMMENTED** [review]
LGTM

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Review by smoshiur1237 — APPROVED** [review]
LGTM

File: scripts/feature_tests/upgrade/upgrade_vars.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ export CAPI_VERSION="${CAPI_VERSION:-v1alpha4}"
 export CAPI_REL_TO_VERSION="v1.0.1"
```
Comment (mboukhalfa) [inline scripts/feature_tests/upgrade/upgrade_vars.sh:10]: should not this be the latest

Comment (furkatgofurov7) [reply scripts/feature_tests/upgrade/upgrade_vars.sh:10]: +1 also [CAPM3_REL_TO_VERSION](https://github.com/metal3-io/metal3-dev-env/blob/main/scripts/feature_tests/upgrade/upgrade_vars.sh#L13) needs to be updated

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: scripts/feature_tests/upgrade/upgrade_vars.sh (hunk 1 above)
Comment (kashifest) [reply scripts/feature_tests/upgrade/upgrade_vars.sh]: Done

**Comment by kashifest** [comment]
@furkatgofurov7 tests have passed

**Review by furkatgofurov7 — APPROVED** [review]
/approve

**Pull request merged**